GeneralisedFilters/src/containers.jl

```diff
@@ -78,22 +91,21 @@ cases through a single-scalar caching mechanism. For standard PF, ll_baseline eq
 LSE before adding observation weights. For APF with resampling, it includes first-stage
 correction terms computed during the APF resampling step.
 """
-function marginalise!(state::ParticleDistribution)
```

Comment:
Is there a reason for passing in both `state` and `particles`. Can we not use `particles = state.particles`?